build.js

```diff
@@ -129,6 +130,21 @@ const modifyBuiltFiles = {
   }
 }
 
+/**
+ * @param {string[]} extensions - The extension of the imported files to exclude. Must match the fill ending path in the import(js) or url(css) statement.
+ * @returns {esbuild.Plugin}
+ */
+const excludeFilesPlugin = (extensions) => ({
+  name: 'exclude-files',
+  setup (build) {
+    build.onResolve({ filter: /.*/ }, async (args) => {
+      if (extensions.some(ext => args.path.endsWith(ext))) {
+        return { path: args.path, namespace: 'exclude', external: true }
```

Comment:
What's the `namespace` property?

Reply:
i don't believe it's needed here, but https://esbuild.github.io/plugins/#namespaces talks more about it. 

Essentially, everything is `file` namespace by default.. but you can change the namespace so different plugins can operate on things. We only have custom plugins here, and none of ours are using that namespace, so we could remove it if we want, but, for example, we could filter on namespaces and then simply have a plugin that adds `external: true` for anything in the `exclude` namespace.. see https://esbuild.github.io/plugins/#on-resolve-options

Reply:
it allows for modularity